benchmarks/templates/benchmarks/table.html

```diff
@@ -93,13 +93,12 @@
                 <td class="score_cell"><strong>{{ model.rank }}</strong></td>
                 <td class="model">
                     <div class="identifier" title="{{ model.reference_identifier }}">
-                        <a target="_self" rel="noopener noreferrer" href="{% url 'model' model.id %}">
+                         <a target="_self" rel="noopener noreferrer" href="{% url 'model' model.id %}">
```

Comment:
undo, the extra space is incorrect formatting indentation (I counted 5 rather than 4 spaces with the change)